projects/invariant/index.js

```diff
@@ -0,0 +1,17 @@
+const retry = require("async-retry");
+const axios = require("axios");
+
+async function fetch() {
+  const response = (
+    await retry(
+      async () => await axios.get("https://stats.invariant.app/short/mainnet")
```

Comment:
How difficult is it to fetch the tvl by querying solana rpc instead? I know we dont have the best support for solana, but if you hop onto our discord, I can help with the adapter change if you are struck somewhere

Reply:
Calculating total tvl is quite complicated since we don't store any stats on-chain.
1. Total tvl is the sum of all pools. Currently there are 22 pools, but we anticipate many more as anyone can add their own pool from UI and it only costs a few dollars.
2. Computing tvl for a single pool is also not trivial - it requires fetching all open positions since the program stores "virtual liquidity" due to a concentrated liquidity mechanism.

Overall all functions to compute total tvl are public, but requires fetching all positions and all pools.

Reply:
I understand, can you revert the change to the template file, I would merge this PR